scalableminds/webknossos
# Use isVisible attribute in backend correctly

### URL of deployed dev instance (used for testing):
- https://___.webknossos.xyz

### Steps to test:
- create invisible tree -> should correctly load invisble in next load
- invisble tree in nml (`color.a = 0` or `isVisible = false`) -> should load tracing with invisble tree

------
